database/migrations/2020_11_16_215145_create_products_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->string('slug');
+            $table->longText('description')->nullable();
+            $table->string('sku');
+            $table->decimal('amount', 10, 2)->nullable();
+            $table->dateTime('publishedAt');
```

Comment:
just a suggestion.. maybe we could do `published_at` here so it comforms to other laravel table naming

Reply:
Good call. I'll fix this. I also need to remove some other stuff since it looks like LitStack has it's own way of using models/controllers/etc. I'll try to fix this today